packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -77,7 +77,11 @@ export const registerStream = (observer, name, cb) => {
 
 const maxRetries = 12;
 
-const notifyRetry = (msg, error, delay) => log(`${msg}: ${error.error.msg_type}, ${localize('retrying in')} ${delay}s`);
+const notifyRetry = (msg, error, delay) => {
+    const extra = {};
+    extra.error_content = `${msg}: ${error.error.msg_type}, ${localize('retrying in')} ${delay}s`;
```

Comment:
Please try to provide as much context to our translators as possible by using placeholders.

```JS
    extra.error_content = `${msg}: ${error.error.msg_type}, ${localize('retrying in')} ${delay}s`;
```

With the above, all the translators will see is `retrying in`, which doesn't mean a lot by itself, so they'll likely reach out to us to provide that extra context. We can avoid this by using placeholders:

```JS
    extra.error_content = localize('{{ message }}: {{ message_type }}, retrying in {{ delay_in_seconds }}s', {
        message: msg,
        message_type: error.error.msg_type,
        delay_in_seconds: delay,
    });
```

With the above, translators will see `{{ message }}: {{ message_type }}, retrying in {{ delay_in_seconds }}s`, which will provide extra context to them.

Please also ensure that `msg` and `error.error.msg_type` are localised wherever they're coming from.